.github/ci-control/should_run_project.sh

```diff
@@ -0,0 +1,79 @@
+# Decides whether we should build a project in the current branch based on the ci-config.json file.
```

Comment:
Not a blocker, but could we pick python or something else thats better than bash? :)

Reply:
i'm afraid to admit that i'm more familiar with bash than python :) And i don't know bash at all :) so maybe i should try to learn python and convert this :)

Reply:
moved to python. Should not review on full stomach.